src/chain_actions/delete_action.rs

```diff
@@ -27,7 +27,7 @@ impl DeleteAction {
         S: 'static + From<crate::signals::ActionSignal<T>> + serde::Serialize + std::fmt::Debug,
         E: 'static,
     {
-        delete_entry(header_hash.clone().into())?;
+        delete_entry(DeleteInput::new(header_hash.clone().into(), ChainTopOrdering::Relaxed))?;
```

Comment:
do we hardcode Relaxed elsewhere? I think so, but just checking. We are just following with that here?

Reply:
yeah we do so I just followed that